<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cVarListClass</class>
 <widget class="QWidget" name="cVarListClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cVarList</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#cVarListClass {
    background-color: #3a3a3a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
    border: 1px solid #505050; /* 与背景协调的边框颜色 */

}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="top" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#top {
    background-color: #4a4a4a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
    border: 1px solid #505050; /* 与背景协调的边框颜色 */

}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLineEdit" name="sourchEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sourch">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #4d4d4d, stop:1 #5e5e5e); /* 更明显的灰色渐变背景 */
    border: 2px solid #666666; /* 更深的灰色边框 */
    border-radius: 10px; /* 圆角 */
    padding: 6px 10px; /* 内边距 */
    color: #ffffff; /* 白色文字，增加对比 */
    font-size: 14px; /* 字体大小 */
    text-align: center; /* 文本居中 */
    box-shadow: 0 4px 8px rgba(0,0,0,0.25); /* 增强的外部阴影，更强立体感 */
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色渐变背景 */
    border: 2px solid #256f25; /* 更深绿色边框 */
    color: #ffffff; /* 白色文字 */
    box-shadow: 0 2px 5px rgba(0,0,0,0.3) inset; /* 内凹阴影，强调按下的立体效果 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #636363, stop:1 #747474); /* 更浅的灰色渐变背景 */
    border: 2px solid #7d7d7d; /* 浅灰色边框 */
    color: #aaaaaa; /* 更浅的灰色文本，增强对比度 */
    box-shadow: none; /* 去除阴影，表示不可操作 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色背景的渐变与按下时一致 */
    border: 2px solid #555555; /* 较暗的灰色边框，表示不可操作 */
    color: #ffffff; /* 白色文本 */
    box-shadow: none; /* 去除阴影，强调不可操作状态 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}</string>
        </property>
        <property name="text">
         <string>搜索</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reset">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #4d4d4d, stop:1 #5e5e5e); /* 更明显的灰色渐变背景 */
    border: 2px solid #666666; /* 更深的灰色边框 */
    border-radius: 10px; /* 圆角 */
    padding: 6px 10px; /* 内边距 */
    color: #ffffff; /* 白色文字，增加对比 */
    font-size: 14px; /* 字体大小 */
    text-align: center; /* 文本居中 */
    box-shadow: 0 4px 8px rgba(0,0,0,0.25); /* 增强的外部阴影，更强立体感 */
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色渐变背景 */
    border: 2px solid #256f25; /* 更深绿色边框 */
    color: #ffffff; /* 白色文字 */
    box-shadow: 0 2px 5px rgba(0,0,0,0.3) inset; /* 内凹阴影，强调按下的立体效果 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #636363, stop:1 #747474); /* 更浅的灰色渐变背景 */
    border: 2px solid #7d7d7d; /* 浅灰色边框 */
    color: #aaaaaa; /* 更浅的灰色文本，增强对比度 */
    box-shadow: none; /* 去除阴影，表示不可操作 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色背景的渐变与按下时一致 */
    border: 2px solid #555555; /* 较暗的灰色边框，表示不可操作 */
    color: #ffffff; /* 白色文本 */
    box-shadow: none; /* 去除阴影，强调不可操作状态 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}</string>
        </property>
        <property name="text">
         <string>复位</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="mid" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#mid {
    background-color: #4a4a4a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
    border: 1px solid #505050; /* 与背景协调的边框颜色 */

}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>列表视图</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QTableWidget" name="varTable"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>树状图</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>1</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>1</number>
          </property>
          <property name="bottomMargin">
           <number>1</number>
          </property>
          <property name="spacing">
           <number>1</number>
          </property>
          <item row="0" column="0">
           <widget class="QTreeWidget" name="varTree">
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="botton" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#botton
 {
    background-color: #4a4a4a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
    border: 1px solid #505050; /* 与背景协调的边框颜色 */

}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="botton0" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#botton0
 {
    background-color: #5a5a5a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
    border: 1px solid #505050; /* 与背景协调的边框颜色 */

}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>275</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="pageInfo">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #4d4d4d, stop:1 #5e5e5e); /* 更明显的灰色渐变背景 */
    border: 2px solid #666666; /* 更深的灰色边框 */
    border-radius: 2px; /* 圆角 */
    padding: 4px 4px; /* 内边距 */
    color: #ffffff; /* 白色文字，增加对比 */
    font-size: 16px; /* 字体大小 */
    text-align: center; /* 文本居中 */
    box-shadow: 0 4px 8px rgba(0,0,0,0.25); /* 增强的外部阴影，更强立体感 */</string>
           </property>
           <property name="text">
            <string>1/50</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pageUp">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #4d4d4d, stop:1 #5e5e5e); /* 更明显的灰色渐变背景 */
    border: 2px solid #666666; /* 更深的灰色边框 */
    border-radius: 10px; /* 圆角 */
    padding: 6px 10px; /* 内边距 */
    color: #ffffff; /* 白色文字，增加对比 */
    font-size: 14px; /* 字体大小 */
    text-align: center; /* 文本居中 */
    box-shadow: 0 4px 8px rgba(0,0,0,0.25); /* 增强的外部阴影，更强立体感 */
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色渐变背景 */
    border: 2px solid #256f25; /* 更深绿色边框 */
    color: #ffffff; /* 白色文字 */
    box-shadow: 0 2px 5px rgba(0,0,0,0.3) inset; /* 内凹阴影，强调按下的立体效果 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #636363, stop:1 #747474); /* 更浅的灰色渐变背景 */
    border: 2px solid #7d7d7d; /* 浅灰色边框 */
    color: #aaaaaa; /* 更浅的灰色文本，增强对比度 */
    box-shadow: none; /* 去除阴影，表示不可操作 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色背景的渐变与按下时一致 */
    border: 2px solid #555555; /* 较暗的灰色边框，表示不可操作 */
    color: #ffffff; /* 白色文本 */
    box-shadow: none; /* 去除阴影，强调不可操作状态 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}</string>
           </property>
           <property name="text">
            <string>上一页</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pageDown">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>-1</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #4d4d4d, stop:1 #5e5e5e); /* 更明显的灰色渐变背景 */
    border: 2px solid #666666; /* 更深的灰色边框 */
    border-radius: 10px; /* 圆角 */
    padding: 6px 10px; /* 内边距 */
    color: #ffffff; /* 白色文字，增加对比 */
    font-size: 14px; /* 字体大小 */
    text-align: center; /* 文本居中 */
    box-shadow: 0 4px 8px rgba(0,0,0,0.25); /* 增强的外部阴影，更强立体感 */
}

QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色渐变背景 */
    border: 2px solid #256f25; /* 更深绿色边框 */
    color: #ffffff; /* 白色文字 */
    box-shadow: 0 2px 5px rgba(0,0,0,0.3) inset; /* 内凹阴影，强调按下的立体效果 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #636363, stop:1 #747474); /* 更浅的灰色渐变背景 */
    border: 2px solid #7d7d7d; /* 浅灰色边框 */
    color: #aaaaaa; /* 更浅的灰色文本，增强对比度 */
    box-shadow: none; /* 去除阴影，表示不可操作 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}

QPushButton:disabled:pressed {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #297229, stop:1 #3cb043); /* 绿色背景的渐变与按下时一致 */
    border: 2px solid #555555; /* 较暗的灰色边框，表示不可操作 */
    color: #ffffff; /* 白色文本 */
    box-shadow: none; /* 去除阴影，强调不可操作状态 */
    border-radius: 10px; /* 圆角 */
    font-size: 14px; /* 字体大小 */
}</string>
           </property>
           <property name="text">
            <string>下一页</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="botton1" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#botton1
 {
    background-color: #5a5a5a; /* 更浅的灰色背景 */
    color: #eeeeee; /* 更亮的灰色字体，提高可读性 */
    font-size: 14px; /* 字体大小适中 */
    border: 1px solid #505050; /* 与背景协调的边框颜色 */

}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>361</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
    stop:0 #4d4d4d, stop:1 #5e5e5e); /* 更明显的灰色渐变背景 */
    border: 2px solid #666666; /* 更深的灰色边框 */
    border-radius: 2px; /* 圆角 */
    padding: 4px 4px; /* 内边距 */
    color: #ffffff; /* 白色文字，增加对比 */
    font-size: 16px; /* 字体大小 */
    text-align: center; /* 文本居中 */
    box-shadow: 0 4px 8px rgba(0,0,0,0.25); /* 增强的外部阴影，更强立体感 */</string>
           </property>
           <property name="text">
            <string>单页显示数量</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="cSuperBtn" name="varPerPage">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cSuperBtn</class>
   <extends>QPushButton</extends>
   <header>include/cwidget/cSuperBtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
